test/components/yDragBoxLayerTests.ts

```diff
@@ -1,23 +1,14 @@
 ///<reference path="../testReference.ts" />
 
-describe("Layer Components", () => {
+describe("SelectionBoxLayer", () => {
```

Comment:
Not sure this makes sense? What are the `SelectionBoxLayer` tests called then?

I notice that the `SelectionBoxLayer` tests are grouped under `"Interactive Components"`; ironically, that's a good grouping title for these tests, but not the `SelectionBoxLayer` ones!